react/components/organisms/listing/listing.stories.js

```diff
@@ -76,10 +76,28 @@ storiesOf("Organisms", module)
                 />
             );
         };
+        const searchingHeaderLayout = select(
+            "Search Header Layout",
+            {
+                Unset: undefined,
+                Horizontal: "horizontal",
+                Vertical: "vertical"
+            },
+            "horizontal"
+        );
+        const expandSearchBar = bool("Expand Search Bar", false);
+        const expandAnimationDuration = number("Expand Animation Duration", 200);
 
         return (
             <View style={{ flex: 1, paddingVertical: 15, backgroundColor: "#f6f7f9" }}>
-                <Listing items={() => getItems()} renderItem={renderItem} filters={filters} />
+                <Listing
+                    items={getItems}
+                    renderItem={renderItem}
+                    filters={filters}
+                    searchingHeaderLayout={searchingHeaderLayout}

```

Comment:
please rename `searching` prefixed variables to `search` as I believe we're already calling it the "search" element and not "searching"
```suggestion
                    searchHeaderLayout={searchingHeaderLayout}

```